pkg/kubectl/cmd/log.go

```diff
@@ -92,71 +81,86 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 		Aliases: []string{"log"},
 	}
-	cmd.Flags().BoolVarP(&o.Follow, "follow", "f", o.Follow, "Specify if the logs should be streamed.")
-	cmd.Flags().BoolVar(&o.Timestamps, "timestamps", o.Timestamps, "Include timestamps on each line in the log output")
-	cmd.Flags().Bool("interactive", true, "If true, prompt the user for input when required. Default true.")
+	cmd.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
```

Comment:
Split the flags that are bound to the `PodLogOptions` into a separate call that takes the FlagSet and returns the `PodLogOptions` ref.